nustyle.css

```diff
@@ -123,6 +123,11 @@ span.signPortal.signPortalKeiji {
   border: none !important;
   background-color: red;
 }
+span.signPortal.signPortalKadai {
+  border: 3px solid blue !important;
```

Comment:
伝え忘れていてごめん、他のバッジのデザイン変更を適用して欲しいです・・！申し訳ない・・。
```suggestion
  border: none !important;
```

Reply:
<img width="939" alt="スクリーンショット 2022-09-28 15 28 24" src="https://user-images.githubusercontent.com/65702927/192704618-c9604ff8-cf9a-4c28-93a6-cdfb01c7e61e.png">
<img width="953" alt="スクリーンショット 2022-09-28 15 28 12" src="https://user-images.githubusercontent.com/65702927/192704629-79bcdd25-e25a-4fbf-a8aa-a69cb873307c.png">
こちらがはみ出したときの画像です。フルサイズで表示すればはみ出ないですが、950px以下になるとはみ出し、☓ ボタンにアクセスできなくなります。

Reply:
なるほど、気づかなかった・・！
ありがとう！！！